openadapt/finetune.py

```diff
@@ -0,0 +1,52 @@
+from openadapt import crud, models
+from loguru import logger
+import fire
+
+VALID_MODELS = ("davinci", "mpt-7b")
+
+
+def main(recording_id: int, model_name: str):
+    finetune_recording = crud.get_recording_by_id(recording_id)
+    if not finetune_recording:
+        raise ValueError("A recording with the given id does not exist")
+
+    if model_name.lower() not in VALID_MODELS:
+        raise ValueError("This model is not supported for finetuning yet")
```

Comment:
```suggestion
    assert model_name.lower() not in VALID_MODELS, "This model is not supported for finetuning yet"
```

Reply:
same thing - line might be too long, correct it if you take this suggestion

Reply:
otherwise, 
```suggestion
    if model_name.lower() not in VALID_MODELS:
        raise NotImplementedError("This model is not supported for finetuning yet")
```